docs/pages/includes/install-linux.mdx

```diff
@@ -181,5 +181,14 @@ Next, use the appropriate commands for your environment to install your package.
 
   </TabItem>
   </Tabs>
+  <Details title="Is my Teleport instance compatible with Teleport Cloud?">
+
+  Before installing a `teleport` binary with a version besides v(=cloud.version=),
```

Comment:
How about we use `cloud.major_version` instead? There's nothing wrong with not being on the exact same patch.

Reply:
```suggestion
  Before installing a `teleport` binary with a version besides v(=cloud.major_version=),
```